document-portal/document-portal.h

```diff
@@ -42,7 +42,7 @@ char ** document_add_full (int                      *fd,
                            int                      *parent_dev,
                            int                      *parent_ino,
                            int                       n_args,
-                           DocumentAddFullFlags      flags,
+                           DocumentAddFullFlags     *flags,
```

Comment:
Small suggestion, since flags is now an array with size n_args, it might be clearer to move it to above the int n_args argument.